src/OrleansRuntime/Core/InsideRuntimeClient.cs

```diff
@@ -434,11 +435,34 @@ private void SafeSendResponse(Message message, object resultObject)
             }
         }
 
+        private static readonly Lazy<Func<Exception, Exception>> prepForRemotingLazy = 
```

Comment:
It provides access to the internal PrepForRemoting method. As with any reflection code on non public members, I prefer that if there is an exception (not that I expect one) , that it is thrown to the callers of this, and that's why I made it lazy, otherwise this results in an undeterministic TypeInitializationException the moment this type is JIT compiled. 
Better to be clear if the unexpected happens.